refinery/config/settings/base.py

```diff
@@ -615,3 +615,10 @@ def get_setting(name, settings=local_settings, default=None):
 
 # temporary feature toggle for using S3 as user data file storage backend
 REFINERY_S3_USER_DATA = get_setting('REFINERY_S3_USER_DATA', default=False)
+
+# ALLOWED_HOSTS required in 1.8.16 to prevent a DNS rebinding attack.
+ALLOWED_HOSTS = get_setting("ALLOWED_HOSTS")  # NOQA: F405
+
+MIGRATION_MODULES = {
+    'chunked_upload': 'dependency_migrations.chunked_upload'
```

Comment:
3rd party app has not added their own migration, so we needed to build and include one. Author said they are getting around to it soon.

Reply:
https://github.com/juliomalegria/django-chunked-upload/pull/21